specification/iothub/resource-manager/Microsoft.Devices/2017-07-01/iothub.json

```diff
@@ -887,7 +894,7 @@
         "tags": [ "Certificates" ],
         "summary": "Get the certificate list.",
         "description": "Returns the list of certificates.",
-        "operationId": "Certificates_ListByIoTHub",
+        "operationId": "Certificates_ListByIotHub",
```

Comment:
Just curious, what issues did you notice with AutoRest due to this.

Reply:
It's kind of funny. If we leave it how it was previously spelled, at least the python generator was giving me the function "list_by_io_thub", changing the casing properly gives me "list_by_iot_hub".

Reply:
I see, I think we are extracting words that start with a capitalized letter or grouping together capitalized letters into a single word, for most cases this is a viable solution while `IoT` looks like an exception. I guess this is a preferred workaround.